internal/resources/metal/virtual_circuit/resource.go

```diff
@@ -111,12 +119,33 @@ func Resource() *schema.Resource {
 				Optional:     true,
 				RequiredWith: []string{"vrf_id"},
 				Description:  "The Metal IP address for the SVI (Switch Virtual Interface) of the VirtualCircuit. Will default to the first usable IP in the subnet.",
+				Computed:     true,
```

Comment:
I think we could either add this and remove `RequiredWith` on line 120, which would make `metal_ip` optional for VRF VCs, or we could leave this attribute as-is (the `RequiredWith` means the user will always be forced to specify a value for `metal_ip` for a VRF VC).

Reply:
Makes sense. I removed the required_with attribute since according to Metal API the `customer_ip` and `metal_ip` params are optional. Only `subnet` is actually required when working with a VRF VC.